test/test_jsinterp.py

```diff
@@ -2,116 +2,68 @@
 
 from __future__ import unicode_literals
 
-# Allow direct execution
```

Comment:
Direct execution is still possible. Accidental removal?

Reply:
Yes, fixed.